src/components/SchedulePage/Schedule.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import TimeSlot from '../SchedulePage/Timeslot';
+
+// Must be ordered
+const testEvents = [
+	{
+		name: 'ACM-W X ACM Hack | Hackathon 101 + Team Formation',
+		location: 'Sycamore Room',
+		// eslint-disable-next-line no-multi-str
+		description: 'First hackathon? We\'ll show you all the tips and tricks you\'ll ' +
```

Comment:
We can replace the apostrophes with right single quotes.
```suggestion
		description: 'First hackathon? We’ll show you all the tips and tricks you’ll ' +
```